cluster/gce/container-linux/configure-helper.sh

```diff
@@ -516,7 +516,9 @@ function start-kubelet {
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
       flags+=" --api-servers=https://${KUBELET_APISERVER}"
       flags+=" --register-schedulable=false"
-      flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
+      if [[ "${ENABLE_MASTER_NOSCHEDULE_TAINT:-false}" == "true" ]]; then
+        flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
```

Comment:
If reverting this is necessary to revert, it will be impossible for us to migrate to taints from unscheduable because that would be an unacceptable breaking change.